tasks/Win32NT/service.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "Copy service file"
+  win_template:
+    src: service.cmd.j2
+    dest: "{{ slave_windows_workdir }}/service.cmd"
+
+- name: "Install nssm"
+  win_chocolatey:
+    name: nssm
+    state: present
+
+- name: "Create a service via nssm"
+  win_nssm:
+    name: "{{ slave_windows_service }}"
+    application: "{{ slave_windows_workdir }}/service.cmd"
```

Comment:
Double quotes are dangerous for using since any backslash in a path is treated like escape symbol. Please, use single quotes.

Reply:
fixed